app/views/batches/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><span class="fa fa-copy"></span> Batch <%= @batch.job_id %></h1>
+
+<div class='table-responsive'>
+  <table class='table table-striped'>
+    <thead>
+      <th scope='col'>Status</th>
+      <th scope='col'>Row Number</th>
+      <th scope='col'>Accession Number</th>
+    </thead>
+    <tbody>
+      <% @batch.batch_items.each do |item| %>
```

Comment:
I think you need some `<tr>`'s here (and above). This is putting all the csv rows on one line.